### PR TITLE
Improvements for price and badge

### PR DESCRIPTION
**Pull request by AlleeX**
### WHAT DOES THIS PR DO: ###

Improvements for price and badge

Fix issue #132

**Price with discount:**

before:

selection_453

after:

selection_452

**badge:**

before:

selection_455

after:

selection_454

### WHERE SHOULD THE REVIEWER START: ###
Diffs

### CODE REVIEW

File: components/badge/badge.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const Badge = ({ arrow, border, children, mods, position, title, visible, ...oth
     badge = visible ? (
       <div {...otherProps} className={getClassNamesWithMods('ui-badge-badge', mods, badgeMods)}>
         {title}
+        {border ? <div className="ui-badge__border" /> : null}
```
Comment (yurist38) [inline components/badge/badge.js]: could be just `{border && <div className="ui-badge__border" />}`

File: components/price/price.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ export function addThousandsSeparator(value, thousandsSeparator) {
     return value;
   }
 
+  console.log(value);
```
Comment (yurist38) [inline components/price/price.js]: Forgotten? 😄

File: tests/unit/badge/badge.spec.js [hunk 3]
```diff
@@ -65,4 +65,12 @@ describe('Badge', () => {
 
     expect(shallowToJson(wrapper)).toMatchSnapshot();
   });
+
+  it('should render badge without border', () => {
+    const wrapper = shallow(
+      <Badge border={false} title="Badge title" />
+    );
+
```
Comment (froskie) [inline tests/unit/badge/badge.spec.js]: Perhaps test the style props?

File: components/badge/badge.js (hunk 1 above)
Comment (AlleeX) [inline components/badge/badge.js]: @yurist38 done. thanks ;)

File: tests/unit/badge/badge.spec.js (hunk 3 above)
Comment (AlleeX) [inline tests/unit/badge/badge.spec.js]: @froskie ?)

**Pull request merged**